### PR TITLE
Added CMakeLists.txt

### PR DESCRIPTION
**Pull request by nasseef**
@abukamail please check this update.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.0)
```
Comment (abukamail) [inline CMakeLists.txt:6]: Add description of the file.

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+set(CMAKE_BUILD_TYPE Debug)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
+
+project(a.out)
+add_executable(
+    a.out
+    rectangle-main.cc
+    rectangle.cc
+)
```
Comment (abukamail) [inline CMakeLists.txt]: Add a blank line.

**Review by abukamail — COMMENTED** [review]
See my code comments.

File: CMakeLists.txt (hunk 1 above)
Comment (nasseef) [reply CMakeLists.txt:6]: Will do. Good idea.

File: CMakeLists.txt (hunk 2 above)
Comment (nasseef) [reply CMakeLists.txt]: A blank line shall be added.

**Pull request merged**